src/com/lilithsthrone/game/dialogue/utils/UtilText.java

```diff
@@ -680,7 +680,7 @@ public static String parse(List<GameCharacter> specialNPC, String input) {
 					}
 				}
 				
-				if (openBrackets>0 && ((target!=null && command!=null) || String.valueOf(c).matches(".") || c!=' ')) {
+				if (openBrackets>0 && ((target!=null && command!=null) || String.valueOf(c).matches(".") || c!=' ' || c!=Character.MIN_VALUE)) {
```

Comment:
This change doesn't seem to work for me locally. If this condition was changed to always exclude non-breaking spaces (`openBrackets>0 && c!='\u200b'`) then it does work for me. Hopefully this isn't a java implementation thing.

As a side note, `c!=' '` is true for all characters except a space. `String.valueOf(c).matches(".")` is true for all characters (except maybe linebreaks, depends on java implementation), but definitely true for a space. (Maybe this was supposed to be `"\\."`? Which then it could be `c=='.'` instead.)

Regardless, `c!=' ' || c!=Character.MIN_VALUE` is equivalent to always true as well.

Therefore, the second clause is always true, and the only condition that matters in the current code is `openBrackets>0`.

Reply:
The Character.MIN_VALUE represent an empty char(or null) because for some reason there's a bunch of them. It's also not a consistent issu sometime there's a bunch of them and sometime it just work's. All its doing is ignoring all the empty char that are present
![inked2018-07-01 23_48_49-lilith s throne 0 2 7 8 alpha not responding _li](https://user-images.githubusercontent.com/33431624/42144576-ba01158e-7d89-11e8-8310-0ca779f25394.jpg)

Reply:
I see this (`'\u200b'` is a zero-width space):
![image](https://user-images.githubusercontent.com/31363213/42146681-d4d65c66-7d8e-11e8-96c5-3cea3c0bed9f.png)